sklearn/linear_model/tests/test_base.py

```diff
@@ -530,11 +540,11 @@ def test_preprocess_data_weighted(is_sparse):
     assert_array_almost_equal(yt, y - expected_y_mean)
```

Comment:
Should we update tests with `assert_allclose` according to reasoning in [#21701](https://github.com/scikit-learn/scikit-learn/pull/21701#discussion_r824542679)?
I checked for one test at random we have a few failures when we replace `assert_array_almost_equal` with `assert_allclose`

Reply:
No. We should keep PRs small and only solve one problem at a time.

Reply:
That's fair